lib/realtime/data/stations.ex

```diff
@@ -56,4 +56,58 @@ defmodule Realtime.Data.Stations do
       "Braintree-02" => "place-brntn"
     }
   end
+
+  @spec next_stations(String.t() | nil, integer()) :: [String.t()]
+  def next_stations(station_id, direction) do
+    case {station_id, direction} do
+      # Southbound
+
+      {"place-alfcl", 0} -> ["place-davis"]
```

Comment:
I had no simple ideas for computing this at runtime, so I did what Glides does and put it together manually. Makes me wonder, though...

Reply:
Haven't internalized GTFS enough to know if it easily represents these station connections? In any case, I think it's fine to hard-code it like this.